Code/Framework/AzTest/AzTest/AzTest.cpp

```diff
@@ -161,6 +161,9 @@ namespace AZ
                 AZ::Test::printUnusedParametersWarning(argc, argv);
                 AZ::Test::addTestEnvironments(m_envs);
                 m_returnCode = RUN_ALL_TESTS();
+                if (::testing::UnitTest::GetInstance()->test_to_run_count() == 0 || ::testing::UnitTest::GetInstance()->successful_test_count() == 0)
```

Comment:
Question about the successful_test_count() == 0, so will the "No tests found" message display if all tests in a suite fail?